src/components/Main.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+
+export default function Main(props) {
+    return (
+        <main className={props.darkMode ? "dark" : ""}>
```

Comment:
since you only have one alternative you can make this a logical AND operator 
```suggestion
        <main className={props.darkMode && "dark"}>
```